src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -0,0 +1,55 @@
+import nifgen
```

Comment:
There should be entries in tox.ini to run these tests on nimi-bot as well as a way to run them explicitly for test suites.

Reply:
Fixed